README.md

```diff
@@ -1,2 +1,149 @@
+
+
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Rails React 👋🌎</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 [Hello Rails React] <a name="about-project"></a>
+
+> About
+A "Hello world" project built with React & ROR as one app with Webpack
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">ReactJS</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux Toolkit</a></li>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/index.html">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Other</summary>
+  <ul>
+    <li><a href="https://docs.github.com/en/get-started/quickstart/github-flow">GitHub flow</a></li>
+    <li><a href="https://rubocop.org/">Rubocop</a></li>
+    <li><a href="https://stylelint.io/">Stylelint</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need :arrow_down: :
+
+- Check the Ruby version or if you have at all by running `ruby -v` if it returns  `ruby 3.1.2p20 (2022-04-12 revision 4491bb740a)` or similar, you have Ruby installed.
+
+* Ruby version used `ruby 3.1.2`
+
+### Setup 🛠️
+
+Clone this repository to your desired folder:
+
+
+- Clone the repository using the GitHub this link [🔗](https://github.com/karlavdelgadof/hello-rails-react.git)
+- `cd hello-rails-react`
+
+### Install 📥
+
+To get this project running correctly, please run the following command to install all necessary dependencies:
+
+- Run `bundle install`
+- Run `npm install`
+- Run `npm run build`
+- Run `yarn install`
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `rails server`
+- Open Browser `http://localhost:3000/`
+- Ensure you have PostgreSQL installed before running the application`psql --version`.
+- Run `rails db:setup`.
+- Create a [.envs](https://medium.com/geekculture/postgresql-rails-and-macos-16248ddcc8ba) file using the link to host postgres credentials.
```

Comment:
- [ ] Let the users know that htey need to run rails db:create and migrate for your migrations
![image](https://user-images.githubusercontent.com/62705947/207640704-89117ac6-aa2a-444b-b5ab-937f4384197c.png)

Reply:
Hi @GzimAsani Thank you for pointing this out :smile: , however, I added the "Run `rails db:setup` which is a task that runs `db:create` `db:migrate` and `db:seeds` :heavy_check_mark:

Reply:
Hi @karlavdelgadof I red the `rails db:setup` as `rails db:seed` :smile:. Sorry

Reply:
No worries! Thank you :smile: